src/config/mod.rs

```diff
@@ -75,13 +75,12 @@ impl From<AdmonitionInfoRaw> for AdmonitionInfo {
             collapsible,
         } = other;
         let (directive, title) = match (Directive::from_str(&raw_directive), title) {
-            (Ok(directive), None) => (directive, ucfirst(&raw_directive)),
```

Comment:
The `ucfirst` functionality is still required to support alternate titles based on the directive given - I've added a regression test in #85 for this to demo why